core/spec/models/spree/payment_spec.rb

```diff
@@ -1231,26 +1246,17 @@
     end
   end
 
-  describe "is_cvv_risky?" do
-    it "returns false if cvv_response_code == 'M'" do
-      payment.update_attribute(:cvv_response_code, "M")
-      expect(payment.is_cvv_risky?).to eq(false)
-    end
-
-    it "returns false if cvv_response_code == nil" do
-      payment.update_attribute(:cvv_response_code, nil)
-      expect(payment.is_cvv_risky?).to eq(false)
-    end
-
-    it "returns false if cvv_response_message == ''" do
-      payment.update_attribute(:cvv_response_message, '')
-      expect(payment.is_cvv_risky?).to eq(false)
+  describe "#is_cvv_risky?" do
+    it "returns false if cvv_response_code is 'M' or nil" do
+      ['M', nil].each do |char|
+        payment.cvv_response_code = char
+        expect(payment.is_cvv_risky?).to eq(false)
+      end
```

Comment:
Nit: I'd prefer separate examples for nil and 'M' like in the original code since if the example right now fails, you'll have to debug the example to see if it failed because the character was "M" or nil.

Reply:
I'll move the block outside of the `it` which should give us the best of both worlds 👍